From 5734e1627de9b6decb4cafaf47121843df1a8d09 Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Mon, 21 Oct 2019 15:25:19 -0400
Subject: [PATCH] Reduce some requirements.

Signed-off-by: Elliott Sales de Andrade <quantum.analyst@gmail.com>
---
 requirements/test.txt | 8 ++++----
 setup.py              | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/requirements/test.txt b/requirements/test.txt
index 6b97bf8..0431793 100644
--- a/requirements/test.txt
+++ b/requirements/test.txt
@@ -1,8 +1,8 @@
-attrs >= 19.1.0
-hypothesis >= 4.24, < 5
+attrs >= 18.2.0
+hypothesis >= 3.66.11, < 5
 Pillow >= 5.0.0
-pytest >= 4.4.0, < 5
-pytest-xdist >= 1.28, <  2
+pytest >= 3.10.1, < 5
+pytest-xdist >= 1.27.0, <  2
 pytest-helpers-namespace >= 2019.1.8
 pytest-timeout >= 1.3.3
 python-xmp-toolkit >= 2.0.1 ; sys_platform != "nt"
diff --git a/setup.py b/setup.py
index bff96aa..120d127 100644
--- a/setup.py
+++ b/setup.py
@@ -147,7 +147,7 @@ if __name__ == '__main__':  # for mp_compile
         setup_requires=[
             'setuptools_scm',
             'setuptools_scm_git_archive',
-            'pybind11 >= 2.4.3, < 3',
+            'pybind11 >= 2.3.0, < 3',
         ],
         use_scm_version=True,
         tests_require=tests_require,
-- 
2.21.0

